src/AutoUI/Lenses/types/entity.tsx

```diff
@@ -21,22 +21,26 @@ export const entity: LensTemplate = {
 			model,
 			autouiContext,
 		}: EntityLensRendererProps<any>) => {
+			// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Can we please add the reasons we're disabling next line in the comments? Just from looking at it I can't tell/remember why, so I'd like to know whenever I visit the code if it's that way because of a hack or with intent